PBS_gadi_runPipeline.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+  
+#PBS -P er01 
+#PBS -N Somatic-shortV
+#PBS -l walltime=02:00:00
+#PBS -l ncpus=1
+#PBS -l mem=60GB
```

Comment:
Is 60gb required to run the head job? I suspect not

Reply:
- The gatk container is a big one. It did crash once due to memory related issues. So changed this value to a bit bigger
- I will test once more